frameworks/cassandra/tests/test_recovery.py

```diff
@@ -129,3 +124,28 @@ def get_pod_host(pod_name):
         if labels[i]['key'] == 'offer_hostname':
             return labels[i]['value']
     return None
+
+def wait_for_all_up_and_normal(pod_host, task_id):
```

Comment:
these variables could be more descriptive, e.g.:
- `pod_host_to_replace`
- `task_exec_task_id`